resources/lib/play.py

```diff
@@ -0,0 +1,103 @@
+import xbmcgui
+import xbmcaddon
+import xbmcplugin
+import comm
+import config
+import sys
+import urllib2
+import urlparse
+import drmhelper
+
+from aussieaddonscommon import utils
+
+_url = sys.argv[0]
+_handle = int(sys.argv[1])
+
+
+def parse_m3u8(data, m3u8_path, qual=-1):
```

Comment:
is this custom, or could we use an m3u8 python module?

Reply:
Yeah I made that from scratch. I tried the m3u8 module that was in plus7 but it would give the wrong quality stream.